### ISSUE TITLE
Create website and fill with content

### ISSUE THREAD

**Issue by kkapsner** [kkapsner/CanvasBlocker#106]
The addon needs a proper website.

**Comment by kkapsner** [comment]
Created (sub-)domain canvasblocker.kkapsner.de - no real content yet.

**Comment by spodermenpls** [comment]
No spelling mistakes this far - I will continue to have a close look at it.

**Comment by jawz101** [comment]
on Android, should I use the one on the new website or the one on AMO?

http://canvasblocker.kkapsner.de/fennec/

https://addons.mozilla.org/en-US/android/addon/canvasblocker/

**Comment by kkapsner** [comment]
You should first try the version from AMO.

**Comment by kkapsner** [comment]
Maybe an interactive translation tool would also be great.

**Comment by spodermenpls** [comment]
"Can't download because there is no download button. If you can't provide one then your product is crap and should go back to the drawing board."

https://www.youtube.com/watch?v=wXHq-FO9Dho

**Comment by kkapsner** [comment]
Yes - sometimes the problem is clearly in front of the screen.

@jawz101: spodermenpls is referring to a review of CB: https://addons.mozilla.org/de/firefox/addon/canvasblocker/reviews/1210803/

**Comment by spodermenpls** [comment]
@kkapsner How do you feel about adding a "FAQ" link/line to the hanger menu? And you could also transfer/duplicate the FAQ section from your homepage to the still empty "Wiki" section here on Github (in between "Security" and "Projects" at the top).

**Comment by kkapsner** [comment]
Adding the FAQ to the browser action page is a good idea.

I do not want to duplicating the FAQ (I'm lazy...) - so I just disabled the "Wiki" section.

**Comment by spodermenpls** [comment]
@kkapsner Since this 2* review is idling on AMO for a while now, maybe give this poor fella a nudge to get over here and file an issue regarding the whitelisting/broken sites, I am curious about what he's referring to (besides the lack of 'Haute Couture' regarding the settings panel): https://addons.mozilla.org/de/firefox/addon/canvasblocker/reviews/1579178/

**Comment by kkapsner** [comment]
Thanks for notifying me. I did not get a mail for this review and was on vacation the last weeks.

**Comment by zero77** [comment]
@kkapsner 
Have you considered, https://pages.github.com

**Comment by kkapsner** [comment]
No, have not yet considered. But I'm not sure if I trust github enough to host my addon update url (for the alpha versions). When the code base for the website increases it will be part of this repository. Synching to my hoster is then not a big deal.

**Comment by spodermenpls** [comment]
@kkapsner Now that I have a repo of my own, I've seen that there's the possibility to add a website to the description section. I think `https://canvasblocker.kkapsner.de` would be a sensible addition here.

**Comment by kkapsner** [comment]
Thanks for the hint. Added the link.

**Comment by spodermenpls** [comment]
@kkapsner Apropos, since your FAQ section on your website still resembles (layout-wise) [motherfuckingwebsite.com](url), you could follow the advice on [bettermotherfuckingwebsite.com](url) and "turn that grotesque pile of shit into an easy-to-read masterpiece", as its author phrased it.

**Comment by kkapsner** [comment]
Hm... making all the pages a little bit more readable is a good idea.

**Comment by Thorin-Oakenpants** [comment]
@spodermenpls that's an awesome link ... I love lightweight. I remember the days of the 5k challenge: to build an entire website (not just one page) in under 5kb

holy *motherfucking* shit ... https://alistapart.com/article/another-10k-apart/  .. yup, 2000 rings a bell - I see they upped it to 10k. The winner in 2000 from memory had multiple items/pages, a shopping cart, and payment/checkout = all fully functional

**Comment by spodermenpls** [comment]
Nice little shoutout from Mr. Rossmann: https://youtu.be/KMLMQRS3Krk?t=943
